scripts/rnnlm/train_rnnlm.sh

```diff
@@ -23,6 +23,8 @@ final_effective_lrate=0.0001
 embedding_l2=0.005
 embedding_lrate_factor=0.1  # the embedding learning rate is the
                             # nnet learning rate times this factor.
+backstitch_scale=0.0        # backstitch training scale
+backstitch_interval=1       # backstitch training interval
 cmd=run.pl  # you might want to set this to queue.pl
```

Comment:
I just noticed this here - @danpovey should we just change the default to queue.pl then?

Reply:
I prefer to always leave the default of cmd at run.pl and have it always passed in from the command line.